modules/react/combobox/lib/Combobox.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import {createContainer} from '@workday/canvas-kit-react/common';
+
+import {useComboboxModel} from './useComboboxModel';
+import {ComboboxInput} from './ComboboxInput';
+import {ComboboxMenu} from './ComboboxMenu';
+export interface ComboboxProps {
+  /**
+   * Children of the Combobox. Should contain a `<Combobox.Input>`, a `<Combobox.Content>`
```

Comment:
```suggestion
   * Children of the Combobox. Should contain a `<Combobox.Input>` and a `<Combobox.Content>`
```

Reply:
content or menu?

Reply:
Menu. This was autogenerated by the `yarn create-component` script